### PR TITLE
Include work items URI in "Sent Helix JOB" message

### PR DESCRIPTION
**Pull request by dougbu**
- #6322

### CODE REVIEW

File: Documentation/AzureDevOps/SendingJobsToHelix.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Tests results for "public" projects are accessible via the link which is provide
 Example build output:
 
 ```Text
-Sent Helix Job 7a6bb019-ed0e-4e46-a065-a38391d90019
+Sent Helix Job ; see work items at https://helix.dot.net/api/jobs/7a6bb019-ed0e-4e46-a065-a38391d90019/workitems?api-version=2019-06-17
 Waiting on job completion...
 Results will be available from https://mc.dot.net/#/user/dotnet-github-anon-kaonashi-bot/pr~2Fdotnet~2Fwinforms~2Frefs~2Fpull~2F9~2Fmerge/type~2Ftests/20181029.7
```
Comment (dougbu) [inline Documentation/AzureDevOps/SendingJobsToHelix.md:137]: Hmm, these lines aren't correct. Should I do the following as well❔
```suggestion
Waiting for completion of job 7a6bb019-ed0e-4e46-a065-a38391d90019 on Windows.10.Amd64.Server20H1.Open
Job 7a6bb019-ed0e-4e46-a065-a38391d90019 on Windows.10.Amd64.Server20H1.Open is completed with 2 finished work items.
```

**Review by ChadNedzlek — APPROVED** [review]

File: Documentation/AzureDevOps/SendingJobsToHelix.md (hunk 1 above)
Comment (dougbu) [reply Documentation/AzureDevOps/SendingJobsToHelix.md:137]: FYI my validation build confirmed the sequence is now. Should I update the entire block❔ The alternative is to wait for a more complete Mission Control eradication I guess.
``` text
Sending Job to Debian.9.Amd64.Arcade.Open...
Sent Helix Job; see work items at https://helix.dot.net/api/jobs/38f272c2-7999-44e4-953b-d5d003b614ce/workitems?api-version=2019-06-17
Waiting for completion of job 38f272c2-7999-44e4-953b-d5d003b614ce on Debian.9.Amd64.Arcade.Open
Job 38f272c2-7999-44e4-953b-d5d003b614ce on Debian.9.Amd64.Arcade.Open is completed with 14 finished work items.
```

Comment (akoeplinger) [reply Documentation/AzureDevOps/SendingJobsToHelix.md:137]: I'd update the doc to the new output.

**Review by akoeplinger — APPROVED** [review]

File: Documentation/AzureDevOps/SendingJobsToHelix.md (hunk 1 above)
Comment (dougbu) [inline Documentation/AzureDevOps/SendingJobsToHelix.md:137]: Complete suggestion
```suggestion
Sending Job to Debian.9.Amd64.Arcade.Open...
Sent Helix Job; see work items at https://helix.dot.net/api/jobs/38f272c2-7999-44e4-953b-d5d003b614ce/workitems?api-version=2019-06-17
Waiting for completion of job 38f272c2-7999-44e4-953b-d5d003b614ce on Debian.9.Amd64.Arcade.Open
Job 38f272c2-7999-44e4-953b-d5d003b614ce on Debian.9.Amd64.Arcade.Open is completed with 14 finished work items.
```

File: Documentation/AzureDevOps/SendingJobsToHelix.md [hunk 2]
```diff
@@ -132,9 +132,11 @@ Tests results for "public" projects are accessible via the link which is provide
 Example build output:
 
 ```Text
-Sent Helix Job 7a6bb019-ed0e-4e46-a065-a38391d90019
-Waiting on job completion...
-Results will be available from https://mc.dot.net/#/user/dotnet-github-anon-kaonashi-bot/pr~2Fdotnet~2Fwinforms~2Frefs~2Fpull~2F9~2Fmerge/type~2Ftests/20181029.7
+Sent Helix Job ; see work items at https://helix.dot.net/api/jobs/7a6bb019-ed0e-4e46-a065-a38391d90019/workitems?api-version=2019-06-17
```
Comment (dougbu) [inline Documentation/AzureDevOps/SendingJobsToHelix.md:135]:
```suggestion
```

**Pull request merged**